src/test/java/org/broadinstitute/hellbender/tools/spark/bwa/BwaSparkIntegrationTest.java

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.hellbender.tools.spark.bwa;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.SamAssertionUtils;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public final class BwaSparkIntegrationTest extends CommandLineProgramTest {
+
+    @Override
+    public String getTestedClassName() {
+        return BwaSpark.class.getSimpleName();
+    }
+
+    @Test
+    public void test() throws Exception {
+        final File expectedSam = new File(getTestDataDir(), "bwa.sam");
+
+        final File ref = getTestFile("ref.fa");
+        final File input = getTestFile("R.bam");
+        final File output = createTempFile("bwa", ".bam");
+        output.delete();
```

Comment:
\[FindBugs\] INFO: RV: Exceptional return value of java\.io\.File\.delete\(\) ignored in org\.broadinstitute\.hellbender\.tools\.spark\.bwa\.BwaSparkIntegrationTest\.test\(\)